### PR TITLE
fix: Express dates uniformly and reference agoric-sdk contents by permalink

### PR DESCRIPTION
**Pull request by gibson042**
* Express full dates as "mmm d, yyyy" (strftime `%b %-d, %Y`), e.g. "Nov 5, 1955".
* Express months as "mmmm yyyy" (strftime `%B %Y`), e.g. "October 1985".

The styles were taken from existing content, but I'd be happy to change to any other unambiguous convention.

### CODE REVIEW

File: main/.vuepress/config.js [hunk 1]
```diff
@@ -45,7 +45,13 @@ module.exports = {
     ]
   ],
 
-  plugins: ['check-md'],
+  plugins: [
+    'check-md',
+    [
+      '@vuepress/last-updated',
+      { dateOptions: { dateStyle: "medium", timeStyle: "long", timeZone: "Etc/UTC" } },
```
Comment (gibson042) [inline main/.vuepress/config.js:52]: This updates the page footers from ambiguous patterns like "_**Last Updated**: 2/20/2022, 1:50:55 AM_" to "_**Last Updated**: Feb 20, 2022, 1:50:55 AM UTC_".

File: main/zoe/guide/contracts/constantProductAMM.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 <Zoe-Version/>
 
-##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/vpool-xyk-amm/multipoolMarketMaker.js) (Last updated: 2021-10-26)
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/15322a1f65bb67e1865f8276600a76171696dc74/packages/zoe/src/contracts/vpool-xyk-amm/multipoolMarketMaker.js) (Last updated: Feb 17, 2022)
```
Comment (gibson042) [inline main/zoe/guide/contracts/constantProductAMM.md:5]: @Chris-Hibbert multipoolMarketMaker.js was moved to the run-protocol package by https://github.com/Agoric/agoric-sdk/commit/a9895c41b57d5ab3543a213824e293ee304dbcaa. This update points to its state immediately before the move, but we could also follow it if we're willing to have a Zoe contracts page link out of the zoe package.

Comment (gibson042) [reply main/zoe/guide/contracts/constantProductAMM.md]: @Chris-Hibbert Done, please verify that the link is correct.

Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/constantProductAMM.md]: I think this is the link 
https://github.com/Agoric/agoric-sdk/blob/363115d47652c43bec8514f624524e70e30f5b8e/packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

Comment (gibson042) [reply main/zoe/guide/contracts/constantProductAMM.md]: And is https://github.com/Agoric/agoric-sdk/blob/4e0aece631d8310c7ab8ef3f46fad8981f64d208/packages/run-protocol/src/vaultFactory/vaultFactory.js correct for the [Treasury contract page](https://github.com/Agoric/documentation/pull/642/files#diff-dd1d62f3cce7d3383e80a0c75a5ee4e38a5e843a708c9863ee44e8aea265d57c)?

Comment (Chris-Hibbert) [reply main/zoe/guide/contracts/constantProductAMM.md]: Yes.

**Pull request merged**